Description: The use of implicit split call has been removed from perl
 in version 5.12 as it had been deprecated for 15 years
Author: Paul Gevers <elbrus@debian.org>
Bug-Debian: http://bugs.debian.org/692342
Forwarded: no

--- a/move3
+++ b/move3
@@ -42,7 +42,7 @@
 }
 
 while (<>) {
-	split;
+	@_=split;
 
 	if ($_[0] eq "D") {
 		if (fileno(MKDIR) == undef) {
